src/library/scala/util/Either.scala

```diff
@@ -506,6 +506,35 @@ object Either {
    */
   def cond[A, B](test: Boolean, right: => B, left: => A): Either[A, B] =
     if (test) Right(right) else Left(left)
+ 
+  /**
+   * Retrieves Left values for several Either values
+   *  {{{
+   *  case class Person(firstName: String, lastName: String, age: Int)
+   *
+   *  def validateAge(intValue: Int): Either[String, Int] = ??? 
+   *  def validateFirstName(stringValue: String): Either[String, String] = ???
+   *  def validateLastName(stringValue: String): Either[String, String] = ???
+   *
+   *  val errors: Seq[String] = Either.lefts(
+   *   validateFirstName("john"),
+   *   validateLastName("doe"),
+   *   validateAge(40)
+   *  )
+   *  if (errors.size) == 0  ???// use for comprehension here to build a Person 
+   *  else ??? // handle errors here
+   *  }}}
+   */
+  def lefts[A](eithers: Either[A, _]*): Seq[A] = eithers.toSeq.collect { 
```

Comment:
I would name it `collectLefts`, what do you think?

Also, the `.toSeq` call should not be necessary.

Reply:
@julienrf indeed, toSeq was not necessary, I've removed it and changed the method names as suggested :)

Thanks
Loïc